AFQ/api/group.py

```diff
@@ -763,6 +763,42 @@ def upload_to_s3(self, s3fs, remote_path):
         if op.exists(self.afqb_path):
             s3fs.put(self.afqb_path, remote_path, recursive=True)
 
+    def export_group_density(self, boolify=True):
+        """
+        Generate a group density map by combining single subject density maps.
+
+        Parameters
+        ----------
+        boolify : bool
+            Whether to turn subject streamline count images into booleans
+            before adding them into the group density map.
+
+        Return
+        ------
+        Path to density nifti file.
+        """
+        densities = self.export("density_maps", collapse=False)
+        ex_density_init = np.load(densities[
+            self.valid_sub_list[0]][
+                self.valid_ses_list[0]])  # for shape
+
+        group_density = np.zeros_like(ex_density_init)
+        self.logger.info("Generating Group Density...")
+        for ii in tqdm(range(len(self.valid_ses_list))):
+            this_sub = self.valid_sub_list[ii]
+            this_ses = self.valid_ses_list[ii]
+            this_density = np.load(densities[this_sub][this_ses])
+            if boolify:
+                this_density = this_density.astype(bool)
+
+            group_density = group_density + this_density
```

Comment:
This sums the densities. Do we want to divide by number of subjects at some point to get the mean density?

Reply:
Yes